flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
this is disappointing

Reply:
even more since it looks like this is written in Java.

Reply:
It's because a dependency (libsignal-client) uses a native library (written in rust), that's unfortunately only included for x86_64.

Reply:
why not building this library from source?